### ISSUE TITLE
Can I train in yolo8 custom dataset the include different size of images? 2048X2048 and 1536X2048?

### ISSUE THREAD

**Issue by roy-orfaig** [ultralytics/ultralytics#19101]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Can I train in yolo8 custom dataset the include different size of images? 2048X2048 and 1536X2048?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @roy-orfaig, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) where you can explore a wealth of [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. These resources may already have insights relevant to your question.

If this is a ❓ Question about custom training with YOLOv8, please provide as much detail as possible. For instance:
- A description of your dataset, including how the annotations are formatted.
- Relevant command-line or Python script snippets.
- Any relevant logs or console outputs.
- Details on the training configuration you intend to use (e.g., epochs, batch size, etc.).

Make sure to check out our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) for guidance on optimizing your training performance.

If this is a 🐛 Bug Report, please share a [minimum reproducible example (MRE)](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug effectively.

### Join the Ultralytics Community
Join the broader Ultralytics community where you can share insights and get advice from other users:
- For real-time chat, visit our [Discord](https://discord.com/invite/ultralytics) 🎧.
- For long-form discussions, head to our [Discourse forum](https://community.ultralytics.com/).
- For community updates and discussions, check out our [Subreddit](https://reddit.com/r/Ultralytics).

### Upgrade to the Latest Version
To ensure optimal results and compatibility, upgrade to the latest `ultralytics` package along with its [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml). Use [**Python>=3.8**](https://www.python.org/) and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/).

```bash
pip install -U ultralytics
```

### Verified Environments
YOLO can be used in various verified environments. Get started quickly by using one of the options below:
- Free GPU Notebooks: 
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Cloud Platforms: [Google Cloud](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) or [AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- Docker: See the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>.

### Repository Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>
If this badge is green, all tests are passing in our CI pipeline, which ensures optimal functionality across platforms.

This is an automated response to assist you quickly 😃. An Ultralytics engineer will review your question in detail and provide a more specific response as soon as possible. Thank you for your patience!

**Comment by Y-T-G** [comment]
Yes
